### ISSUE TITLE
18MS: IC cannot trace route into New Orleans after phase 6

### ISSUE THREAD

**Issue by perwestling** [tobymao/18xx#1898]
Got this report from Mark Derrick: 

"Today in a 3P 18MS game number12722, I could not trace a route into New Orleans from Jackson Mississippi using either of IC's trains. Again, Frisco had a token there and to work around, I had to ask the Frisco player to trace my route into New Orleans using master mode. I refreshed my browser window and cleared all of my trains but to no avail. I am using Chrome mostly. I had no problem tracing into New Orleans earlier in the game with non-permanent trains. This problem seemed to only come up after phase six (permanent trains) had started. As a reminder, this has come up before for me when playing 18MS on the main site. I noticed the OR number in bold today and this should help folks keep up with the ORs better. Thanks for your help with this."

**Comment by Schmel924** [comment]
Well, i cant enter the 18xx.games site for last couple of hours, so i wonder how can you make such a request less than hour ago?

**Comment by perwestling** [comment]
@Schmel924 There is no connection really between 18xx.games site and github.com.

**Comment by perwestling** [comment]
@Schmel924 This was actually reported to me via BGG two days ago, so the "today" is 2020-10-15.

**Comment by roseundy** [comment]
I should probably take a look at this in case it's related to some of my recent routing changes.

**Comment by roseundy** [comment]
I can't reproduce this. I loaded it from action 397 and was able to route into NO just fine.

**Comment by tobymao** [comment]
i wonder if it's some state you have to get into? closing for now

**Comment by perwestling** [comment]
I got this from Mark Derrick earlier this week:

"Incidentally, sometime within the last week, I played 18MS on the site and the bug that I reported happened again. In it, a railroad that does not have a token in New Orleans is unable to include it in a route using a non-permanent train. I believe this only happens after permanent trains have been purchased. In both cases, the player who had a token in New Orleans was able to enter master mode and select the route for the other player. I need to get you more player details and submit an official bug report I guess although I would think this would happen fairly often in games with other players."

So need to look into it again.
